src/main/java/school/lemon/changerequest/java/introduction/hw2/Task1.java

```diff
@@ -2,6 +2,9 @@
 
 public class Task1 {
     public static String makeTags(String tag, String text) {
-        return "";
+
+        String result = String.format("<%s>%s</%s>", tag, text, tag);
```

Comment:
Please read `String.format` docs more careful - there is no need for third parameter. You can re-use parameters within template